src/pages/Lend/LoanChannelCategoryClimateExperiment.vue

```diff
@@ -191,14 +204,24 @@ export default {
 			// Get page limit and offset
 			const limit = 9;
 			const offset = 0;
-			return preFetchChannel(
-				client,
-				// Access map directly since SSR doesn't have mixins available
-				loanChannelQueryMapMixin.data().loanChannelQueryMap,
-				targetedLoanChannelURL,
-				// Build loanQueryVars since SSR doesn't have same context
-				{ ids: [...secondaryEcoLoanChannelIds, targetedLoanChannelID], limit, offset }
-			);
+
+			const promisesArrayOfSecondaryChannels = secondaryEcoLoanChannelIds.map(channelId => {
+				return preFetchChannel(
+					client,
+					loanChannelQueryMapMixin.data().loanChannelQueryMap,
+					loanChannelQueryMapMixin.data().loanChannelQueryMap.find(channel => channel.id === channelId)?.url,
+					{ ids: [channelId], limit, offset }
```

Comment:
I think if you merge in main, you'll have some variables already present or imported to pass the `origin` param.

Reply:
> I think if you merge in main, you'll have some variables already present or imported to pass the `origin` param.

Ok will do!